app/frontend/src/features/auth/password/ChangePasswordPage.tsx

```diff
@@ -82,8 +82,9 @@ export default function ChangePasswordPage() {
             </Alert>
           )}
           <Typography className={classes.infoText} variant="body1">
-            Please enter a new password, or leave the "New password" and
-            "Confirm password" fields blank to unset your password.
+            {accountInfo?.hasPassword
+              ? 'Please enter a password, or leave the "New password" and "Confirm password" fields blank to unset your password.'
+              : "Leave the fields blank to unset your password."}
```

Comment:
```suggestion
              : "Please enter a password."}
```

Reply:
I did think after I committed (and was too late to turn back on my laptop 💤 ) if it makes any sense to clear your password if you don't have one already 😂